14_contains/solution/contains-solution.js

```diff
@@ -0,0 +1,12 @@
+const contains = function(current, search) {
+  if (Object.values(current).includes(search)) return true;
+
+  for (const nestedObject of Object.values(current).filter((value) => typeof value === 'object')) {
+    return contains(nestedObject, search)
+  }
+  
+  return false
+};
```

Comment:
You learn something new everyday! Your changes definitely improve the readability. 

I think it's quite nice that the explicit `NaN` check is not required, since `includes` handles that for us gracefully. So how about keeping the original idea and adding a comment that explains this detail and explains that usually an explicit `NaN` check would be required? 

Also, yeah I'm going to paste the changes into the `.spec` files once it looks like the tests won't change anymore

Reply:
I think that sounds reasonable